spec/features/user_sees_new_message_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.